polymer.json

```diff
@@ -9,6 +9,7 @@
     "src/pages/schedule-page.html",
     "src/pages/speakers-page.html",
     "src/pages/team-page.html",
+    "src/pages/attendees-page.html",
```

Comment:
@zhentian-wan This was the missing part

Reply:
OK, nice 👍 I guess give me the whole night, I might not be able to find it :) thanks @sherry